schemas/pulse-job.yml

```diff
@@ -170,9 +185,10 @@ properties:
       maxLength: 50
       pattern: "^[A-Za-z0-9_-]+$"
 
-  who:
-    title: "who"
-    format: "email"
+  owner:
```

Comment:
as the description denotes, this won't always be an email.  so leaving free-form text for now.